crucible-mir/src/Mir/Trans.hs

```diff
@@ -359,6 +391,26 @@ staticPlace did = do
             MirPlace (G.globalType gv) <$> globalMirRef gv <*> pure NoMeta
         Nothing -> mirFail $ "cannot find static variable " ++ fmt did
 
+-- variant of staticPlace for slices
+-- tpr is the element type; len is the length
+staticPlaces :: HasCallStack => C.TypeRepr tp -> Int -> M.DefId -> MirGenerator h s ret (MirPlace s)
+staticPlaces tpr len did = do
```

Comment:
Perhaps rename this to something like "`staticSlicePlace`" to reflect its intended purpose.

Reply:
It got that way because originally it was different from ```staticPlace``` specifically by handling a length instead of a single element... but it's not that simple any more. Updated in 3444637b455edf15c14963403aa69cd2f4192f5e